### PR TITLE
Fiche film

### PR DESCRIPTION
**Pull request by BenZen59**

### CODE REVIEW

File: src/components/FicheFilm/FicheFilm.css [hunk 1]
```diff
@@ -1,25 +1,68 @@
+body {
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.css]: That body color broke the general background when its assembled with the main page

File: src/components/FicheFilm/FicheFilm.css [hunk 2]
```diff
@@ -1,25 +1,68 @@
...
+  z-index: -1;
+  width: 100%;
+  height: 518px;
+  position: absolute;
+  opacity: 0.5;
+}
+
+.backgroundPoster > img
+{
+  -webkit-box-shadow: inset 0px 2px 50px 13px #000000;
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.css]: double code for the same effect

File: src/components/FicheFilm/FicheFilm.css [hunk 3]
```diff
@@ -59,32 +115,84 @@ img {
...
+  margin: 40px;
 }
 
 .actors > div p {
   font-size: 17px;
-  -webkit-box-shadow: 6px 5px 17px 1px #000;
-  box-shadow: 6px 5px 17px 1px #000;
-  border: 1px solid black;
+  background: #000;
+  background: -moz-linear-gradient(top, rgb(0, 0, 0, 0.5) 0%, #eee8ec 20%, #fff 100%);
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.css:124]: some double code

File: src/components/FicheFilm/FicheFilm.css [hunk 4]
```diff
@@ -59,32 +115,84 @@ img {
...
+  background: -webkit-linear-gradient(top, rgb(0, 0, 0, 0.5) 0%, #eee8ec 20%, #fff 100%);
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.css:125]: same

File: src/components/FicheFilm/FicheFilm.css [hunk 5]
```diff
@@ -59,32 +115,84 @@ img {
...
-  padding: 4px;
-  padding-top: 12px;
+  margin-bottom: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  width: 225px;
+}
+
+.actors > div img:hover {
+  -webkit-transform: scale(1.31);
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.css:139]: Same four line who does the same thing

File: src/components/FicheFilm/FicheFilm.jsx [hunk 6]
```diff
@@ -59,66 +68,94 @@ export default function FicheFilm() {
...
-      </div>
-      <div className="directorTimeDate">
-        <p>
-          {director.job} : {director.name}
+        <p className="vote">
+          <img className="starScore" src={UsersScorePictures} alt="StarScore" />
           <br />
-          <br />
-          {runtime} min ({releaseDate})
+          <div className="numberVote">{numberVote} votes</div>
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.jsx]: div cant be in a p

File: src/components/FicheFilm/FicheFilm.jsx [hunk 7]
```diff
@@ -37,9 +48,7 @@ export default function FicheFilm() {
       )
       .then(({ data }) => {
         setDirector(data.crew[5]);
-        setActor1(data.cast[0]);
-        setActor2(data.cast[1]);
-        setActor3(data.cast[2]);
+        setActors(data.cast);
       })
       .catch(() => {
         console.error('Erreur API');
```
Comment (BBriset) [inline src/components/FicheFilm/FicheFilm.jsx]: no console on dev branch

**Review by BBriset — CHANGES_REQUESTED** [review]
some fix to do :)

File: src/components/FicheFilm/FicheFilm.css (hunk 2 above)
Comment (BenZen59) [reply src/components/FicheFilm/FicheFilm.css]: Double to adapt for all bowser

File: src/components/FicheFilm/FicheFilm.css (hunk 3 above)
Comment (BenZen59) [reply src/components/FicheFilm/FicheFilm.css:124]: Double to adapt for all bowser

File: src/components/FicheFilm/FicheFilm.css (hunk 4 above)
Comment (BenZen59) [reply src/components/FicheFilm/FicheFilm.css:125]: Double to adapt for all bowser

File: src/components/FicheFilm/FicheFilm.css (hunk 5 above)
Comment (BenZen59) [reply src/components/FicheFilm/FicheFilm.css:139]: Same

File: src/components/FicheFilm/FicheFilm.jsx (hunk 7 above)
Comment (BenZen59) [reply src/components/FicheFilm/FicheFilm.jsx]: fix later

File: src/components/FicheFilm/FicheFilm.css (hunk 1 above)
Comment (BenZen59) [reply src/components/FicheFilm/FicheFilm.css]: I retire this

Comment (BBriset) [reply src/components/FicheFilm/FicheFilm.css]: ok

**Review by BBriset — DISMISSED** [review]
ok to me

File: src/components/FicheFilm/FicheFilm.jsx [hunk 8]
```diff
@@ -1,19 +1,27 @@
...
+import buttonTrailer from './play.png';
+import star1 from './star1.png';
+import star2 from './star2.png';
+import star3 from './star3.png';
+import star4 from './star4.png';
+import star5 from './star5.png';
+import buttonAdd from './add.png';
 
 export default function FicheFilm() {
   const [title, setTitle] = useState([]);
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: If your state is an array then it must be on plural. Plz do it from line 13 to 22.

File: src/components/FicheFilm/FicheFilm.jsx [hunk 9]
```diff
@@ -53,72 +59,97 @@ export default function FicheFilm() {
...
       });
   }, []);
 
-  const urlPoster = `https://image.tmdb.org/t/p/original${poster}`;
-  const urlActor1 = `https://image.tmdb.org/t/p/original${actor1.profile_path}`;
-  const urlActor2 = `https://image.tmdb.org/t/p/original${actor2.profile_path}`;
-  const urlActor3 = `https://image.tmdb.org/t/p/original${actor3.profile_path}`;
-  const urlVideo = `https://www.youtube.com/embed/${trailer.key}`;
+  let UsersScorePictures;
+  if (voteAverage === 10) {
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: You can do better and cleaner code here

File: src/components/FicheFilm/FicheFilm.jsx [hunk 10]
```diff
@@ -53,72 +59,97 @@ export default function FicheFilm() {
...
+  const min = (runtime / 60 - hour) * 60;
 
   return (
     <div>
       <div className="titreVote">
         <p>{title}</p>
-        <p className="vote">Users vote : {voteAverage}</p>
+        <div className="vote">
+          <img className="starScore" src={UsersScorePictures} alt="StarScore" />
+          <br />
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx]: No *br* use another method.

**Review by Hacene010 — CHANGES_REQUESTED** [review]
The component is very long. when you code, make the components as short as possible. 
And don't use br : /

**Review by Othmane-Khiter — APPROVED** [review]

File: src/components/FicheFilm/FicheFilm.jsx [hunk 11]
```diff
@@ -58,56 +58,58 @@ export default function FicheFilm() {
...
   return (
     <div>
       <div className="titreVote">
-        <p>{title}</p>
+        <p>{titles}</p>
         <div className="vote">
-          <img className="starScore" src={UsersScorePictures} alt="StarScore" />
-          <br />
-          <div className="numberVote">{numberVote} votes</div>
+          <img className="starScore" src={Star()} alt="StarScore" />
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.jsx:89]: I think you don't need to put the ''() " to Star.

**Review by Hacene010 — APPROVED** [review]
Seems ok for me

**Pull request merged**